picard/ui/coverartbox.py

```diff
@@ -33,7 +33,7 @@ class ActiveLabel(QtGui.QLabel):
     """Clickable QLabel."""
 
     clicked = QtCore.pyqtSignal()
-    imageDropped = QtCore.pyqtSignal(QtCore.QUrl)
+    imageDropped = QtCore.pyqtSignal(QtCore.QUrl, QtCore.QByteArray)
```

Comment:
Sorry, I'm so used to Qt code and I saw camel case being used in this file a lot, so I didn't notice snake-case is preferred in picard. Btw, note that I didn't change the name of that variable, and that is actually a signal, so I'm not sure if it falls under the category of "pre-generated PyQt functions". Do you really think it should use snake-case ?